LOAN/LoanContracts/LoanCore.rdf

```diff
@@ -633,14 +573,14 @@
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-ln-ln;hasOriginatorCompany"/>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-ln;hasOriginatingServiceProvider"/>
```

Comment:
I thought we were trying to avoid including the names of range classes in properties. Why not hasOriginator?

Reply:
In mortgage lending, the originator often refers to the individual banker, so I did not want to change the meaning of what MU had - if you look at the ontology, he distinguished between originating company and person for that reason.  The definitions need work, as do property names, and if we follow some of our other patterns, we could have 3 properties - hasOriginator, hasOriginatingServiceProvider, and hasOriginatingIndividual or something like that - which includes the license.    The terminology here needs work, and should be moved to a US specific ontology, but we'll have to account for the fact that originator often refers to the licensed individual.